Reqnroll/CucumberMessages/Configuration/ICucumberMessagesConfiguration.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Reqnroll.CucumberMessages.Configuration
+{
+    public interface ICucumberMessagesConfiguration
+    {
+        bool Enabled { get; }
+        string BaseDirectory { get; }
+        string OutputDirectory { get; }
+        string OutputFileName { get; }
+        IDGenerationStyle IDGenerationStyle { get; }
```

Comment:
The capitalization of words starting with `ID` is pretty hard and they look bad in JSON. So my suggestion would be to rename this to `IdentityStyle`.

Reply:
Eliminated IDGenerationStyle from Configuration.